lib/ssl/src/tls_handshake_1_3.erl

```diff
@@ -902,6 +921,9 @@ do_wait_ee(#encrypted_extensions{extensions = Extensions}, State0) ->
     {Ref, Maybe} = maybe(),
 
     try
+        %% RFC 6066: handle received/expected maximum fragment length
+        Maybe(maybe_max_fragment_length(Extensions, State0)),
+
```

Comment:
This a comment to follow up if the presence of `max_fragment_length ` in `EncryptedExtensions` affects TLS 1.3 session resumption interoperability with openssl. @IngelaAndin